<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editemailconf</class>
 <widget class="QDialog" name="Editemailconf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configurações</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Projects/Blink/icons/resources.qrc">
    <normaloff>:/png/marca-m.png</normaloff>:/png/marca-m.png</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Lblusername">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuário</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LnEdtusername">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>5</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Lblpassword">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>45</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LnEdtpassword">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Lblserver">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Servidor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LnEdtserver">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>75</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>smtp.gmail.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lblport">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>115</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Porta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LnEdtport">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>465</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="topmenu">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>801</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/png/barra-superior-ok.jpg);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelStatus">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>541</width>
      <height>39</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Conta de e-mail</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PshBtnCancel">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>220</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PshBtnSave">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Salvar</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LnEdtusername</tabstop>
  <tabstop>LnEdtpassword</tabstop>
  <tabstop>LnEdtserver</tabstop>
  <tabstop>LnEdtport</tabstop>
  <tabstop>PshBtnCancel</tabstop>
  <tabstop>PshBtnSave</tabstop>
 </tabstops>
 <resources>
  <include location="../../Projects/Blink/icons/resources.qrc"/>
  <include location="../../Projects/Blink/icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
